app/rdf/parse-search-results.ts

```diff
@@ -105,8 +105,10 @@ function buildSearchCubes(
           const dim = bySubjectAndPredicate.get(x.object.value);
           return {
             iri: joinIris({
-              // We don't need to unversion the cube iri here, as search cubes are
-              // temporary and dimensions coming from here aren't stored in chart config.
+              // TODO
```

Comment:
Comment to see if the TODO has been done :)

Reply:
Yeah, maybe this should be a `MAYBE TODO` :) I am not sure (also from performance reasons) if we should fetch additional triple and unversion cube iri while knowing it's not really needed.

I know it would be a "correct" approach, but I am not 100% sure if it's worth pursuing, what do you think?

Reply:
Ah yes OK I see. I agree, then I think you can remove the TODO and include the "performance reason" bit. 🙇